modular_zubbers/code/datums/greyscale/config_types/greyscale_configs/greyscale_clothes.dm

```diff
@@ -0,0 +1,9 @@
+/datum/greyscale_config/cso
+	name = "command stripper outfit"
+	icon_file = 'modular_zubbers/icons/obj/clothing/under/captain.dmi'
+	json_config = 'code/datums/greyscale/json_configs/cso.json'
```

Comment:
Change this and repath the file to `modular_zubbers`

Reply:
Which line, all of it currently works fine as of my testing.

Reply:
I don't see why it would, It's an individual file that's only called on by the already modularised file, affecting only a bubbers modular item. I'm not sure if it will work outside of that specific folder the way GaGs looks to be coded.

Reply:
It's mostly for the sake of it being easier to find

Reply:
Should be done, testing it shortly.